.github/workflows/sync.yml

```diff
@@ -1,39 +1,27 @@
-# Based on https://www.growingwiththeweb.com/2016/07/enabling-pull-requests-on-github-wikis.html
+# Roughly based on
+#   https://www.growingwiththeweb.com/2016/07/enabling-pull-requests-on-github-wikis.html
+
 name: Sync Two Wiki Repos
 
 on:
-  # Merged PRs
-  push:
-    branches: [master]
-  # Weekly
-  schedule:
-    # https://crontab.guru/#0_12_*_*_1
-    - cron: "0 12 * * 1"
-  # Whenever someone asks
-  workflow_dispatch:
+  push: {branches: [master]}
+  schedule: [{cron: "7 0 * * 1,3"}] # https://crontab.guru/#7_0_*_*_1,3
+  workflow_dispatch:                # on request
 
 jobs:
   sync:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-
-        # Setup Git
-      - run: git config user.name "typescript-bot"
-      - run: git config user.email "bot@typescriptlang.org"
-
-        # Replace remotes with auth'd versions, then pull the .wiki version (the one people can't PR to)
-        # then merge in any changes from here and push
-      - 
-        run: |
-          git remote remove origin
-          git remote add origin https://$GITHUB_TOKEN@github.com/microsoft/TypeScript-wiki.git
-          git remote add upstream https://$GITHUB_TOKEN@github.com/microsoft/TypeScript.wiki.git
-          git fetch origin
-          git fetch upstream
-          git merge upstream/master --no-edit 
-          git push origin HEAD:master
-          git push upstream HEAD:master
-        env:
-          GITHUB_TOKEN: ${{ secrets.TS_BOT_TOKEN }}
```

Comment:
This used to be doing the work, but I can't find where it does something like that that now?  As the 'Run sync' seems to refer to this file ( `./.github/workflows/sync`  )

Reply:
Bah...  I did just the changes by mistake -- it's added now.